### PR TITLE
Add all Dapr x Azure Developer CLI samples to the gallery

### LINKED ISSUE #102: Add all Dapr x Azure Developer CLI samples to the gallery
https://github.com/Azure-Samples?q=dapr+azd-templates&type=all&language=&sort=

### PR DESCRIPTION
**Pull request by hemarina**
fix #102

### CODE REVIEW

**Review by rajeshkamal5050 — APPROVED** [review]

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -309,7 +309,66 @@ const Users: User[] = [
     source: 'https://github.com/tonybaloney/simple-flask-azd',
     tags: ['python'],
   },
-
+  {
+    title: 'Bindings Dapr Csharp Cron PostgreSQL',
```
Comment (savannahostrowski) [inline website/src/data/users.tsx]: Can we use C# instead of Csharp to keep things consistent?

**Review by savannahostrowski — CHANGES_REQUESTED** [review]
One nit!

**Comment by savannahostrowski** [comment]
@greenie-msft tagging in case you want to take a look!

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -309,7 +309,66 @@ const Users: User[] = [
...
-
+  {
+    title: 'Bindings Dapr C# Cron PostgreSQL',
+    description:
+      'Create microservice to demonstrate Dapr\'s bindings API to work with external systems as inputs and outputs. The service listens to input binding events from a system CRON and then outputs the contents of local data to a postgreSQL output binding.',
+      preview: require('./images/bindings-dapr-cron-postgres.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/bindings-dapr-csharp-cron-postgres',
+    tags: ['dapr','csharp','azuredb-postgreSQL','dotnet']
```
Comment (paulyuk) [inline website/src/data/users.tsx]: Should we add azure-containerapps for all of these Dapr ones?

**Review by paulyuk — CHANGES_REQUESTED** [review]
See comments pls

File: website/src/data/users.tsx [hunk 3]
```diff
@@ -309,7 +309,66 @@ const Users: User[] = [
     source: 'https://github.com/tonybaloney/simple-flask-azd',
     tags: ['python'],
   },
-
+  {
+    title: 'Bindings Dapr C# Cron PostgreSQL',
```
Comment (paulyuk) [inline website/src/data/users.tsx]:
```suggestion
    title: 'Microservices App - Dapr Bindings Cron C# ACA PostgreSQL',
```

File: website/src/data/users.tsx [hunk 4]
```diff
@@ -309,7 +309,66 @@ const Users: User[] = [
...
+    description:
+      'Create microservice to demonstrate Dapr\'s bindings API to work with external systems as inputs and outputs. The service listens to input binding events from a system CRON and then outputs the contents of local data to a postgreSQL output binding.',
+      preview: require('./images/bindings-dapr-cron-postgres.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/bindings-dapr-csharp-cron-postgres',
+    tags: ['dapr','csharp','azuredb-postgreSQL','dotnet']
+  },
+  {
+    title: 'Bindings Dapr Python Cron PostgreSQL',
```
Comment (paulyuk) [inline website/src/data/users.tsx]:
```suggestion
    title: 'Microservices App - Dapr Bindings Cron Python ACA PostgreSQL',
```

File: website/src/data/users.tsx [hunk 5]
```diff
@@ -309,7 +309,66 @@ const Users: User[] = [
...
+    description:
+      'Create a microservice app to demonstrate Dapr\'s bindings API to work with external systems as inputs and outputs. The service listens to input binding events from a system CRON and then outputs the contents of local data to a postgreSQL output binding.',
+      preview: require('./images/bindings-dapr-cron-postgres.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/bindings-dapr-python-cron-postgres',
+    tags: ['dapr','python','azuredb-postgreSQL']
+  },
+  {
+    title: 'Bindings Dapr Node.js Cron PostgreSQL',
```
Comment (paulyuk) [inline website/src/data/users.tsx]:
```suggestion
    title: 'Microservices App - Dapr Bindings Cron Node.js ACA PostgreSQL,
```

File: website/src/data/users.tsx [hunk 6]
```diff
@@ -309,7 +309,66 @@ const Users: User[] = [
...
+    description:
+      'Create a microservice app to demonstrate Dapr\'s bindings API to work with external systems as inputs and outputs. The service listens to input binding events from a system CRON and then outputs the contents of local data to a postgreSQL output binding.',
+      preview: require('./images/bindings-dapr-cron-postgres.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/bindings-dapr-nodejs-cron-postgres',
+    tags: ['dapr','nodejs','azuredb-postgreSQL','javascript']
+  },
+  {
+    title: 'Service Invoke Dapr Node.js',
```
Comment (paulyuk) [inline website/src/data/users.tsx]:
```suggestion
    title: 'Microservices App - Dapr Service Invoke Node.js ACA',
```

File: website/src/data/users.tsx [hunk 7]
```diff
@@ -309,7 +309,66 @@ const Users: User[] = [
...
+    description:
+      'Create two microservices that communicate using Dapr\'s Service Invocation API. The Service Invocation API enables your applications to communicate reliably and securely by leveraging auto-mTLS and built-in retries.',
+      preview: require('./images/svc-invoke-dapr.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/svc-invoke-dapr-nodejs',
+    tags: ['dapr','nodejs']
+  },
+  {
+    title: 'Service Invoke Dapr Python',
```
Comment (paulyuk) [inline website/src/data/users.tsx]:
```suggestion
    title: 'Microservices App - Dapr Service Invoke Python ACA',
```

File: website/src/data/users.tsx [hunk 8]
```diff
@@ -309,7 +309,66 @@ const Users: User[] = [
...
+    description:
+      'Create two microservices that communicate using Dapr\'s Service Invocation API. The Service Invocation API enables your applications to communicate reliably and securely by leveraging auto-mTLS and built-in retries.',
+      preview: require('./images/svc-invoke-dapr.png'),
+      website: 'https://github.com/Azure-Samples',
+      author: 'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/svc-invoke-dapr-python',
+    tags: ['dapr','python']
+  },
+  {
+    title: 'Service Invoke Dapr C#',
```
Comment (paulyuk) [inline website/src/data/users.tsx]:
```suggestion
    title: 'Microservices App - Dapr Service Invoke C# ACA',
```

**Review by paulyuk — CHANGES_REQUESTED** [review]

File: website/src/data/tags.tsx [hunk 9]
```diff
@@ -385,5 +386,10 @@ export const Tags: {[type in TagType]: Tag} = {
         description: 'Template architecture uses Azure API Management',
         color: 'hotpink',
     },
+    azurecontainerapp: {
```
Comment (savannahostrowski) [inline website/src/data/tags.tsx]: I think this should be Azure Container Apps.

Here and in the label.

**Review by savannahostrowski — APPROVED** [review]

File: website/src/data/users.tsx [hunk 10]
```diff
@@ -207,58 +196,128 @@ const Users: User[] = [
...
-    title: 'Python (Django) Web App with PostgreSQL',
+    title: 'Microservices App - Dapr Bindings Cron C# ACA PostgreSQL',
     description:
-      'A restaurants review web app that uses whitenoise for static assets and PostgreSQL for the database. Ready for deployment on App Service with a PostgreSQL Flexible Server inside a VNet.',
-      preview: require('./images/msdocs-django-postgresql-sample-app.png'),
-    website: 'https://github.com/Azure-Samples',
-    author: 'Azure Content Team',
-    source: 'https://github.com/Azure-Samples/msdocs-django-postgresql-sample-app',
-    tags: ['python','django', 'webapps', 'azuredb-postgreSQL', 'appservice','vnets'],
+      'Create microservice to demonstrate Dapr\'s bindings API to work with external systems as inputs and outputs. The service listens to input binding events from a system CRON and then outputs the contents of local data to a postgreSQL output binding.',
```
Comment (jongio) [inline website/src/data/users.tsx]:
```suggestion
      'Create microservice to demonstrate Dapr\'s bindings API to work with external systems as inputs and outputs. The service listens to input binding events from a system CRON and then outputs the contents of local data to a PostgreSQL output binding.',
```

**Pull request merged**